coreutils/Makefile

```diff
@@ -9,6 +9,9 @@ COREUTILS_DIR=${CURDIR}/coreutils-6.10
 COREUTILS_LLVM_DIR=${COREUTILS_DIR}/obj-llvm
 COREUTILS_GCOV_DIR=${COREUTILS_DIR}/obj-gcov
 COREUTILS_SRC=${COREUTILS_DIR}/src
+CFLAGS_LLBMC_COREUTILS=-g -I${COREUTILS_LLVM_DIR}/lib -I${COREUTILS_SRC} -I/${COREUTILS_DIR}/lib
+LLBMC_OUTPUT_DIR=${CURDIR}/LLBMC_OUTPUT
+LLBMC_OPTIONS=--ignore-missing-function-bodies -function-name=main -max-loop-iterations=5 -max-function-call-depth=10
```

Comment:
I think -max-loop-iterations=5 is fine, as 10 as I suggested was just a random guess.